src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { uuid } from 'uuidv4';
+import { UserInfo } from '../UsersInfo';
+import { TodoInfo } from '../TodoInfo';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './TodoList.scss';
+
+interface Props {
+  todos: Todo[];
+  users: User[];
+  addTodo: (todo: Todo) => void;
+}
+
+interface State {
+  title: string;
+  name: string;
+  isTitle: boolean;
+  isName: boolean;
+}
+
+export class TodoList extends React.Component<Props, State> {
+  state = {
+    title: '',
+    name: '',
+    isTitle: false,
+    isName: false,
+  };
+
+  clearState = () => {
+    this.setState({
+      title: '',
+      name: '',
+      isTitle: false,
+      isName: false,
+    });
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>) => {
+    const { name, value } = event.target;
+    const key: keyof State = name as keyof State;
+
+    this.setState({
+      [key]: value,
+    } as Pick<State, 'title' | 'name'>);
+  };
+
+  handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const { users, todos } = this.props;
+    const { name, title } = this.state;
+
+    if (!title) {
+      this.setState({
+        isTitle: true,
+      });
+    }
+
+    if (!name) {
+      this.setState({
+        isTitle: true,
+      });
+    }
+
+    const maxId = Math.max.apply(null, todos.map(todo => todo.id)) + 1;
```

Comment:
It seems that uuid is redundant here since you're using Math.max.apply to find the biggest id among the objects.